docs_src/OVMS/camera_serial_number.md

```diff
@@ -4,16 +4,16 @@ Do the following to get the serial number of an Intel® RealSense™ Camera:
 
 1. Build the RealSense version of dlstreamer Docker image if not done yet:
 
-```bash
-    make build-dlstreamer-realsense
-```
+   ```bash
```

Comment:
Formatwise, will this render ok on the docs html? last time i had to fix this formatting issue due to some extra indentation

Reply:
I think so. The indent here makes it so that the code block is actually nested and indented under the numbering. Otherwise it is left justified.